Source/Core/Common/StringUtil.cpp

```diff
@@ -664,3 +665,19 @@ std::vector<std::string> CommandLineToUtf8Argv(const wchar_t* command_line)
   return argv;
 }
 #endif
+
+std::string GetEscapedHtml(std::string html)
+{
+  static constexpr std::array<std::array<const char*, 2>, 5> replacements{
+      // Escape ampersand first to avoid escaping the ampersands in other replacements
+      std::array<const char*, 2>{"&", "&amp;"},  std::array<const char*, 2>{"<", "&lt;"},
```

Comment:
You should be able to compress these down to just the type name like:

```suggestion
      std::array{"&", "&amp;"},  std::array{"<", "&lt;"},
```

Reply:
I tried to do this but freebsd and macOS wouldn't let me.

Reply:
Ah, both freebsd and macOS probably have old compilers installed on the buildbots then